app/models/container_image_registry.rb

```diff
@@ -4,4 +4,10 @@ class ContainerImageRegistry < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
   has_many :containers, :through => :container_images
+
+  virtual_column :host_port, :type => :string
+
+  def host_port
+    port.nil? ? host : host + ":" + port
```

Comment:
> (unrelated to this patch) this statement makes me wonder why the port is defined in db as string and no as a number...

@abonas At the time I found it a whole lot easier to work with strings when were just reflecting attributes off of k8s. The typing made things difficult since some comparisons were done with strings and some with numbers. I'm not sure this still applies.